src/icon-manifest/manifestMerger.ts

```diff
@@ -8,18 +8,18 @@ export function mergeConfig(
   supportedFolders: models.IFolderCollection,
   iconGenerator: models.IIconGenerator): models.IIconSchema {
 
-  const dFiles = customFiles ? customFiles.default : null;
-  const dFolders = customFolders ? customFolders.default : null;
-  const sFiles = customFiles ? customFiles.supported : null;
-  const sFolders = customFolders ? customFolders.supported : null;
+  const defaultCustomFiles = customFiles ? customFiles.default : null;
```

Comment:
@robertohuertasm I changed the names so they are more clear on what they reference.